### ISSUE TITLE
List of patients whose records need completing

### ISSUE THREAD

**Issue by AmaniKrayemRCPCH** [rcpch/rcpch-audit-engine#797]
Users have requested a way to list all patients whose records need completing (specifically those in the submitting cohort, and it's been >1yr since first paediatric assessment). 

On the old platform, they'd get a weekly notification email with a list of patients in the current cohort that had a) incomplete first year of care forms and b) it's been a year since the first assessment

![Image](https://github.com/rcpch/rcpch-audit-engine/assets/126471383/f31ce17e-aa4a-4e71-8fba-162ec1f44eca)

Maybe this could be a card on the organisation view?

**Comment by dc2007git** [comment]
Hi @AmaniKrayemRCPCH, potentially one way we could go about this is by creating a filter functionality in the cases search. So, the user would be able to select a cohort of their choosing, and filter by those cases that aren't yet complete. @eatyourpeas , what do you think?

**Comment by AmaniKrayemRCPCH** [comment]
Agree with the filter functionality for cohort and completeness! 
May also be useful to filter by patients whose First Paediatric Assessments were >1yr ago, if that's feasible

**Comment by eatyourpeas** [comment]
These are all feasible of course. The cases as it stands each have a progress bar below the buttons which gives an immediate summary of how many fields have been scored. This is not filtered by cohort, but by looking at the submission date you can tell. Apart from Croydon, most centres don't have that many cases so they should be able to tell quite easily as it stands which cases are outstanding.

Perhaps we are close to a place where project board can evaluate where we are with the platform, what is good and what needs changing, and then building a list of items which are desirable and some sense of desirable time frame for each.

**Comment by dc2007git** [comment]
I agree, at the sprint we can discuss which tasks/feature requests are priorities. Like Simon said, all the cases do have a progress bar, and they also have an icon (blue circle icon to the side of each case entry) indicating if the data isn't complete.

Having had a look @AmaniKrayemRCPCH , at the moment we can sort by Cohort which would be quite useful for the users. So at least right now they could sort by cohort and refer to that little icon to get to grips with which users in a specific cohort haven't fully been audited yet.

**Comment by eatyourpeas** [comment]
this ideally should be merged with #878

**Comment by mbarton** [comment]
Will be implemented as part of #878
